pkg/connector/ble/ble.go

```diff
@@ -83,8 +91,12 @@ func (c *Connection) flush() bool {
 }
 
 func (c *Connection) Close() {
-	_ = c.client.ClearSubscriptions()
-	_ = c.client.CancelConnection()
+	if err := c.rxChar.EnableNotifications(nil); err != nil {
```

Comment:
Upstream bug causes this to always return an error on MacOS: https://github.com/tinygo-org/bluetooth/blob/v0.11.0/gattc_darwin.go#L215-L217